src/sdk.ts

```diff
@@ -256,6 +256,72 @@ export class WarpSdk {
 
     return this.wallet.tx(txPayload);
   }
+
+  public async depositAccount(sender: string, account: string, token: Token, amount: string): Promise<TxInfo> {
```

Comment:
i'm using `string` as type of `amount` here where web app use `u<Big>`. 
this is the only difference between it and logic in web app

Reply:
maybe i should use `Uint128` here? `reward` type is `Uint128`

Reply:
it's fine, Uint128 = string